HashMap.java

```diff
@@ -1 +1,11 @@
+package com.company.newMap;
+
```

Comment:
Sofia, hi. I'm lost in my old code. And now I have a new code, i wrote it in holidays. I think it's perfect code :)